resources/kcp/charts/kyma-environment-broker/templates/environments-cleanup-job.yaml

```diff
@@ -1,47 +1,48 @@
 {{if eq .Values.global.kyma_environment_broker.environmentsCleanup.enabled true}}
+{{- range $index, $job := .Values.setupJobs }}
 apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
-  name: "kcp-kyma-environments-cleanup"
-  namespace: kcp-system
+  name: "{{$job.name}}"
+  namespace: "{{$job.namespace}}"
```

Comment:
nit: I think we actually want to run these in the same namespace `kcp-system`. Also looks like other jobs don't have namespace preference at all and only this one has it currently defaulted.
```
$ grep resources/kcp/charts/kyma-environment-broker/templates/*job*.yaml -nre 'namespace:'
resources/kcp/charts/kyma-environment-broker/templates/environments-cleanup-job.yaml:6:  namespace: kcp-system
```